src/EPPlusTest/FormulaParsing/Excel/Functions/Engineering/ImRealTest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OfficeOpenXml;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EPPlusTest.FormulaParsing.Excel.Functions.Engineering
+{
+    [TestClass]
+    public class ImRealTest
```

Comment:
Add a few more tests. For example a real part decimal number. An incorrect argument. A value not containing a rea number or imag part like "i2" or "3.2"